File: Linguagem_C/Algoritmos/LogicaParaDesenvolvimentoDeProgramacaoDeComputadores/capitulo_3/exercicio3H.c
#include <stdio.h>

float calculaVolumeCaixaRetangular(float comprimento, float altura, float largura){
  return comprimento * largura * altura;
}

int main(void){
  float comprimento, altura,largura;

  //lê valores
  printf("Digite o comprimento da caixa:");
  scanf("%f",&comprimento);
  printf("Digite a altura da caixa:");
  scanf("%f",&altura);
  printf("Digite a largura da caixa:");
  scanf("%f",&largura);

  printf("O volume da caixa eh: %.2fL\n",calculaVolumeCaixaRetangular(comprimento,altura,largura));
  return 0;
}

File: Python/Algoritmos/LogicaParaDesenvolvimentoDeComputadores/Capitulo_3/exercicio3H.py
"""
Elabor um programa que calcule o volume de uma caixa retangular
"""

def calculaVolume(comprimento,largura,altura):
    return comprimento * largura * altura

comprimento = float(input("Digite o comprimento da caixa:"))
largura = float(input("Digite a largura da caixa:"))
altura = float(input("Digite a altura da caixa:"))

volume = calculaVolume(comprimento,largura,altura)

print(f"O volume da caixa eh: {volume}")
